src/poll.ts

```diff
@@ -206,6 +206,59 @@ namespace Poll
 			return null;
 		}
 
+		let server: DiscordJS.Guild = message.guild;
+
+		let target: DiscordJS.GuildMember = null;
+
+		for (let member of server.members.array())
+		{
+			if ("<@" + member.user.id + '>' === command[1] || "<@!" + member.user.id + '>' === command[1])
+			{
+				target = member;
+
+				if (member.hasPermission("ADMINISTRATOR") && options.biasAdmin)
+				{
+					message.reply("No can do, all praise " + command[1] + '!');
+					return null;
+				}
+			}
+		}
+
+		if (target === null)
+		{
+			message.reply("No user found by " + command[1] + '.');
+
+			return null;
+		}
+
+		return new Poll
+		(
+			message,
+			desc + ' ' + command[1],
+
+			() => action(target),
+			() => (true),
+			() => Math.floor(server.members.array().length * fraction)
+		);
+	}
+	
+	export function voicePoll
```

Comment:
Instead of having a separate function for voice polls, why not take a boolean as a parameter to turn the voice channel checking on or off? This way we don't duplicate code.

Reply:
originally I had no idea how functions worked here, but I understand it. Will do that